oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/service/ServiceTraffic.java

```diff
@@ -110,16 +136,20 @@ public ServiceTraffic storage2Entity(final Map<String, Object> dbMap) {
         @Override
         public Map<String, Object> entity2Storage(final ServiceTraffic storageData) {
             final String serviceName = storageData.getName();
-            if (NodeType.Normal.equals(storageData.getNodeType())) {
```

Comment:
The normal service condition should be kept. Because no matter what service name of a conjectured service is, it is conjectured by the collected telemetry data. `Group name` should be set by users only.